website/assets/styles/pages/osquery-table-details.less

```diff
@@ -0,0 +1,412 @@
+#osquery-table-details {
+  hr {
+    margin: 0;
+    width: 100vw;
+    height: 1px;
+    border-top: 1px solid @border-lt-gray;
+  }
+  [purpose='search-and-filters'] {
+    [purpose='filter-row'] {
+      max-width: 1200px;
+      padding-left: 40px;
+      padding-right: 40px;
+    }
+    [purpose='platform-selector'] {
+      width: 160px;
+      height: 40px;
+      border-radius: 8px;
+      padding: 9px 16px;
+      background: #FAFAFA;
```

Comment:
👋 hi @eashaw, I'm helping Mike review this.

Would consider just using color variables all around instead of a mix of variables + hex codes; I'm seeing some hex codes that are already in `/assets/styles/mixins-and-variables/colors.less`, so might be easier to keep track of them all that way

Reply:
Thank you! I've updated the stylesheet to use variables from colors.less